include/fastdds/rtps/writer/RTPSWriter.h

```diff
@@ -194,6 +194,20 @@ class RTPSWriter
      */
     RTPS_DllAPI virtual const fastdds::rtps::IReaderDataFilter* reader_data_filter() const = 0;
 
+    /**
+     * @brief Check if a specific change has been delivered to the transport layer of every matched remote RTPSReader
+     * at least once.
+     *
+     * @param change Pointer to the CacheChange to check.
```

Comment:
This is making the documentation build fail.

```suggestion
     * @param seq_num Sequence number of the change to check.
```